src/main/java/gameframework/motion/MoveStrategyKeyboard.java

```diff
@@ -74,22 +77,23 @@ public void keyReleased(int keyCode) {
 	}
 
 	public void goRight() {
-		speedVector.setDirection(new Point(1, 0));
+		speedVector = new SpeedVector(new Point(1, 0));
```

Comment:
After quick reflexion, I've no real use with this code but I don't understand why the speed is always at the default speed even if the Movable element stay in place. 
But I can use the default code in my case so I will close the request.